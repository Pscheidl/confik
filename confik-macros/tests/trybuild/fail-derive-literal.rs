#[derive(confik::Configuration)]
#[confik(derive("hello world"))]
struct A;

fn main() {}
