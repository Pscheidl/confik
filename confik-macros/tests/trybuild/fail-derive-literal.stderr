error: Expected a path to a derivable trait, got Lit(Lit::Str { token: "hello world" })
 --> tests/trybuild/fail-derive-literal.rs:2:17
  |
2 | #[confik(derive("hello world"))]
  |                 ^^^^^^^^^^^^^
